client/src/components/svg/ListeningIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
we can probably get rid of this and use the mic icon here https://lucide.dev/icons/mic

and then make it red for recording

Reply:
![image](https://github.com/danny-avila/LibreChat/assets/11950213/4a22a342-c72d-4fce-976e-2084aaf76661)